src/systems/GameSystem.cpp

```diff
@@ -57,6 +137,49 @@ namespace indie
                     component->getCurrentFrame() = 0;
             }
         }
+        // static int i = 0;
+        // static int j = 0;
+
+        // i++;
+        // if (i % 3 == 0) {
+        //     auto components = sceneManager.getCurrentScene()[IEntity::Tags::SPRITE_2D][1]->getFilteredComponents({ IComponent::Type::RECT });
+        //     auto r = Component::castComponent<Rect>(components[0]);
+        //     r->left = r->width * j;
+        //     if (++j > 5)
+        //         j = 0;
+        // }
+        // if (i == 100) {
+        //     std::shared_ptr<Entity> entity = std::make_shared<Entity>();
+        //     std::shared_ptr<Position> component = std::make_shared<Position>(500, 100);
+        //     std::shared_ptr<Sprite> component4 = std::make_shared<Sprite>("test_pictures/raylib_logo.png");
+        //     entity->addComponent(component).addComponent(component4);
+        //     sceneManager.getCurrentScene().addEntity(entity);
+        // } else if (i == 200) {
+        //     sceneManager.getCurrentScene().removeEntity(sceneManager.getCurrentScene()[IEntity::Tags::SPRITE_2D][2]);
+        // }
+        // auto component = Component::castComponent<ModelAnim>((*sceneManager.getCurrentScene()[IEntity::Tags::RENDERABLE_3D][0])[IComponent::Type::ANIMATION]);
+        // component->getCurrentFrame()++;
+        // if (component->getCurrentFrame() >= component->getNbFrames())
+        //     component->getCurrentFrame() = 0;
```

Comment:
why is this in the comment section ?

Reply:
I let some comments because this is some tests from the others people so i let it to let them test if they want